annotation_widgets/event_validation/widget.py

```diff
@@ -1,14 +1,40 @@
-
-
-from .logic import EventValidationLogic
-from .io import EventValidationIO
-from annotation_widgets.widget import AbstractAnnotationWidget
 import tkinter as tk
 
+from annotation_widgets.widget import AbstractAnnotationWidget
 from models import ProjectData
+from .gui import EventValidationStatusBar, EventValidationSideBar
+from .io import EventValidationIO
+from .logic import EventValidationLogic
+from ..image.canvas import BaseCanvasView
```

Comment:
Try to not import classses from `image` package. 
If you need BaseCanvasView here it can be moved to `annotation_widgets/gui.py`